source/microbit/main.cpp

```diff
@@ -166,11 +166,11 @@ int main(void) {
         // mode.  If we are in "raw REPL" mode then this will be skipped.
         if (pyexec_mode_kind == PYEXEC_MODE_FRIENDLY_REPL) {
             file_descriptor_obj *main_module;
-            if (APPENDED_SCRIPT->header[0] == 'M' && APPENDED_SCRIPT->header[1] == 'P') {
+            if (main_module = microbit_file_open("main.py", 7, false, false)) {
```

Comment:
Please use extra parenthesis around this expression, like it was done below.

Reply:
Done.